ml-agents/mlagents/trainers/stats.py

```diff
@@ -95,8 +108,8 @@ def write_stats(
     ) -> None:
         for val, stats_summary in values.items():
             set_gauge(
-                GaugeWriter.sanitize_string(f"{category}.{val}.mean"),
-                float(stats_summary.mean),
+                GaugeWriter.sanitize_string(f"{category}.{val}.aggregated_value"),
```

Comment:
This is going to break some of our internal processes (that live outside of this repo).

Can you make it:
```
set_gauge(GaugeWriter.sanitize_string(f"{category}.{val}.mean"), float(stats_summary.mean))
set_gauge(GaugeWriter.sanitize_string(f"{category}.{val}.sum"), float(stats_summary.sum))
```
instead?